src/components/form-elements/_checkbox-radio.scss

```diff
@@ -1,5 +1,4 @@
-$crInputSize: fontSize(default);
-$crInputFontSize: fontSize(default);
+$crInputSize: rhythm(2 / 3);
```

Comment:
I think this is better as we get full units (sometimes units like 15.98px were breaking layout for me)